lib/l10n/app_en.arb

```diff
@@ -17,5 +17,101 @@
   "contribute": "Contribute",
   "teachers": "Teachers",
   "my_groups": "My Groups",
-  "login": "Login"
+  "login": "Login",
+  "@about": {},
+  "terms_of_service": "Terms Of Service",
+  "privacy_policy": "Privacy Policy",
+  "@about_view": {},
+  "loading_contributors": "Loading Contributors ...",
+  "about_title": "ABOUT",
+  "about_subtitle": "Learn about the Awesome people behind CircuitVerse",
+  "about_description": "CircuitVerse is a product developed by students at IIIT-Bangalore. It aims to provide a platform where circuits can be designed and simulated using a graphical user interface. While users can design complete CPU implementations within the simulator, the software is designed primarily for educational use. CircuitVerse is an opensource project with an active community. Checkout the contribute page for more detail.",
+  "email_us_at": "Email us at",
+  "join_slack": "Join and chat with us at",
+  "slack_channel": "Slack channel",
+  "contributors": "Contributors",
+  "contributors_subtitle": "Meet the awesome people of CircuitVerse community that've made this platform what it is now.",
+  "@about_tos_view": {},
+  "tos_section1_title": "1. User Agreement",
+  "tos_section1_text1_string1": "1.1 These Terms of Use constitute an agreement between you and the CircuitVerse Team that governs your use of ",
```

Comment:
Is it possible to have a nested section for TOS and Privacy Policy ? Right now it's looking little messy and soon after localizing more stuff, this file will be nightmare to read. That's why I feel some organizing is needed.

Reply:
Can you elaborate on what the nested section means for the current context?.

Here It is important to know where the string belongs, so what I done is 
1. create a resource using module name as the name for companion message ```eg. @about```
2. Now string which repeated in the same module and in different dart files can be put directly below ```@about``` companion message
3. Now for each dart file we use again companion message in name of the file eg. ```@about_view```

In this way, we follow the minimum requirements and give structure to translations as well.

general template::

@Module name
   strings repeated throughout the module
@dart file1
   string resource for the file1
@dart file 2
   string resource for the file 2

Reply:
I meant nested section in such a way:
```
TOS_Section: {
 tos_string_1: value,
 tos_string_2: value
}
```

The template that we are following seems fine to me. If you think the current implementation is just perfect then it's fine by me, I just had this suggestion if we can make it more neat and clean by nesting approach.

Reply:
I agree with you, I tried this as its the basic thing required for the i18n! but there is a limitation with ```arb files``` and their specification doesnt give capability of nesting the objects/keys inside one another. 

actually intl while converting into dart code just generate getters and assign them to the strings.

Minimally we can change setup and start using custom delegates which can map nested strings somehow but this will be very complicated.

In my opinion minimal and efficient setup can be this only!

Reply:
I think this shall be fine! In Simulator the library we use can't fetch nested strings, so we kept things simple! 

I think major amount of libraries for javascript and android apps are designed considering no major data required for translation!

https://github.com/localizely/intl_utils/issues/40

https://www.applanga.com/docs/formats/arb

Reply:
Alright then, it's fine